configs/benchmark/templates/dashboard.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: wrk2
+    grafana_dashboard: "true"
+  name: grafana-dashborad
+  namespace: {{.Release.Namespace}}
+data:
+{{ (.Files.Glob "dashboards/grafana-wrk2-cockpit.json").AsConfig | indent 2 }}
```

Comment:
This leads to a dashboard which I cannot modify (saving the modifications fails with an error message). Not sure if we wan this at this point, as we will need to iterate on the dashboard still. (e.g. the service mesh resource consumption isn't displayed yet)

Also, the dashboard is deployed every time i kick off a new benchmark, creating a new dashboard (and deleting the old one) so I cannot watch successive benchmarks on the same dashboard (always have to find the "new" one in Grafana first). This was not the case with the upload/download script that was using the Grafana API - re-uploading existing dashboards did not create a new one; instead, the current one was updated.

Reply:
Another point is we don't want dashboard here(benchmarking cluster). It is to be deployed on the OC. So I will keep it separate for now.